.gitignore

```
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
```

У вас закоммичена папка `.idea`. Она не должна попадать в гит, так как она не имеет ничего общего с проектом. Вам нужно либо настроить глобальный гитигнор, например, как сказано тут - https://gist.github.com/subfuzion/db7f57fff2fb6998a16c (код для мака подходит и для линукса), либо нужно добавить папку сюда
```suggestion
/.idea
```